support-lib/jni/djinni_support.cpp

```diff
@@ -371,6 +371,15 @@ jstring jniStringFromUTF8(JNIEnv * env, const std::string & str) {
     return res;
 }
 
+jstring jniStringFromWString(JNIEnv * env, const std::wstring & str) {
+
+    std::u16string utf16(str.begin(), str.end());;
```

Comment:
It doesn't really matter what the size of `wchar_t` is if it can be assigned to `char16_t` without error. But later down `env->NewString()` expects Unicode characters which is not guaranteed to be the native encoding of `wchar_t`/`wstring`.